.travis.yml

```diff
@@ -17,8 +17,6 @@ before_install:
   - export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
   - pip install urllib3[secure]
   - pip install gsutil --user
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
```

Comment:
I always wondered in the back of my head *why*

Reply:
It has existed since the beginning of the project (88c9720fde66be37febd027267820880792f1d16)